gateway/apicast.d/ssl.conf

```diff
@@ -0,0 +1,9 @@
+listen 8443 ssl;
+
+ssl_certificate {{ prefix }}/conf/serving-cert/tls.crt;
```

Comment:
Unfortunately those files can't be templated by liquid yet as they are also included in plain nginx in our test suite. We are making a workaround in https://github.com/3scale/apicast/pull/610 so we could apply something similar here.